src/traversable/defaultOptions.ts

```diff
@@ -0,0 +1,57 @@
+import { XMLNode } from '../XMLNode';
+
+const utf8Decoder = new TextDecoder();
+
+export const decoder = {
+  decode: (array: Uint8Array) => {
+    return utf8Decoder.decode(array);
+  },
+};
+export interface OptionsType {
```

Comment:
Please rename it to ParseOptions and this should also be the name of the file

Reply:
done